tron/api/resource.py

```diff
@@ -321,17 +321,13 @@ def get_config_index(self):
     @AsyncResource.bounded
     def render_GET(self, request):
         config_name = requestargs.get_string(request, 'name')
-        no_header = requestargs.get_bool(request, 'no_header')
```

Comment:
I think you need to leave this for backwards-compatability and remove it in a later change. paasta_tools currently passes this param: https://github.com/Yelp/paasta/blob/master/paasta_tools/tron/client.py#L89

It can do nothing, but the API needs to accept that param without breaking.

Reply:
Fair.

Reply:
Oops, I think you're right. No need to change, then.